server/repository/migrations/postgres/2022-06-21-013225_create_log_table/up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TYPE log_type AS ENUM (
+    'USER_LOGGED_IN',
+    'INVOICE_CREATED',
+    'INVOICE_STATUS_ALLOCATED',
+    'INVOICE_STATUS_PICKED',
+    'INVOICE_STATUS_SHIPPED',
+    'INVOICE_STATUS_DELIVERED',
+    'INVOICE_STATUS_VERIFIED'
+);
+
+CREATE TABLE log (
+    id TEXT NOT NULL PRIMARY KEY,
+    type log_type, 
+    user_id TEXT,
+    store_id TEXT REFERENCES store(id),
+    record_id TEXT REFERENCES invoice(id),
```

Comment:
Do want this reference to invoice(id) here? Will this table be used for other non-invoice records?